src/web/auth.rs

```diff
@@ -156,22 +156,28 @@ impl FromRequest<ServerState> for HawkPayload {
             80
         };
 
+        let path = request
+            .uri()
+            .path_and_query()
+            .ok_or(HawkErrorKind::MissingPath)?;
+        let expiry = if path.path().ends_with("/info/collections") {
```

Comment:
I was going to suggest a stricter check by looking into the request.resource() -- the ResourceDef's pattern inside of that -- but the resource method appears to be removed in actix-web 1.

I wonder if there's another way to determine what route triggered the request in 1.0

Reply:
Good question! Looking at the path like this does feel a bit hacky, but it kinda makes sense. We're fetching the path of the request URI, which returns a string that we're doing a comparison on. I don't know if there's a more reliable way of fetching the HTTP request element, but I suspect it's probably private.

Reply:
No current 1.0 way of getting that resource def (but maybe in the future): https://github.com/actix/actix-web/issues/911